test/suites/Tests-8-ECMA.js

```diff
@@ -66,11 +66,13 @@ define(['pumascript', 'esprima'], function(puma, esprima) {
      });
 
      test("configurable", function(){
```

Comment:
Great! I was asking the reason seems clear to me. Another thing is now you will need to rebase because Alan changes were before once this is ready we can merge.

Thanks!!